package.json

```
@@ -15,8 +15,8 @@
   },
   "exports": {
     ".": {
-      "import": "./index.mjs",
-      "default": "./index.js"
+      "import": "./src/index.mjs",
+      "default": "./src/index.mjs"
```

```suggestion
      "import": "./dist/unfetch.mjs",
      "default": "./dist/unfetch.js"
```

Needs to point at the built output, and `default` needs to point to a CJS output module, not ESM.